tools/extend/README.md

```diff
@@ -0,0 +1,42 @@
+# `check` CLI tool
+
+This binary is a thin wrapper around the `rtmr` library to
+extend the measurement into RTMR registers.
+
+The tool's input is the event log.
+
+The tool's output is an error or "Success".
+
+## Usage
```

Comment:
The rtmr flag is used in the example but not listed / described over here.

Reply:
Added